playbooks/support/on-call-slo.md

```diff
@@ -0,0 +1,13 @@
+# On-Call SLO
+
+This document describes a "Service-Level Objective" for the hours during with engineers are expected to be monitoring for incidents during their on-call schedule.
```

Comment:
I believe the proper grammar here is "during which"?

Reply:
also, thoughts on linking to Google's resource on SLOs (https://landing.google.com/sre/sre-book/chapters/service-level-objectives/) considering this term isn't widely used at Artsy yet?

Reply:
Love these clarifications so far! Thanks so much for writing these up! :heart: 

I might rephrase this line to focus on the response time. Here we're setting expectations around when on-call engineers should be available to respond to incidents, but the objectives themselves might be better set as explicit response time thresholds.

For example:
- Incidents should have a response time of under 10m, during hours of full support
- Incidents should have a response time of under 30m, during hours of partial support

We might also define what it means to initially respond to an incident and how that time is measured as part of the SLO. To represent our current process, we might say that to respond the incident, you acknowledge it in the #incidents slack channel and create a Jira Ops incident ticket, transitioned to an active state.

These SLOs  would target the time between first reported in #incidents and a Jira Ops ticket created and transitioned to an active status by an engineer.